src/main/java/com/bakdata/common_kafka_streams/KafkaStreamsApplication.java

```diff
@@ -271,6 +282,16 @@ protected DeleteTopicsResult deleteTopic(final String topic) {
         }
     }
 
+    private void resetSchemaRegistry() throws IOException, RestClientException {
+        final SchemaRegistryClient client = new CachedSchemaRegistryClient(this.schemaRegistryUrl, 100);
+        for (final String type : List.of("-key", "-value")) {
+            for (final String subject : this.inputTopics) {
```

Comment:
the producing application is responsible for schemas of input topics. Thus, you can remove this part